custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js

```diff
@@ -213,6 +221,7 @@ function LocationFilterController($rootScope, $scope, $location, $uibModal, loca
             templateUrl: 'locationModalContent.html',
             controller: LocationModalController,
             controllerAs: '$ctrl',
+            backdrop: vm.isAwcReport() ? 'static' : true,
```

Comment:
what does this do?

Reply:
@calellowitz The ```backdrop``` option in the angular-ui modal turn off the possibility to close the modal when you click outside of it. 

"
backdrop (Type: boolean|string, Default: true) - Controls presence of a backdrop. Allowed values: true (default), false (no backdrop), 'static' (disables modal closing by click on the backdrop).
" - https://angular-ui.github.io/bootstrap/#!#modal

Reply:
thanks!